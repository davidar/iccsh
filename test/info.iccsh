:declare foo_bar_42 const int foo_bar_42 = 42
foo_bar_42
