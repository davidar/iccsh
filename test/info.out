const int foo_bar_42
